www/csp-violation-report-endpoint.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+// Start configure
+$log_file = dirname(__FILE__) . '/csp-violations.log';
```

Comment:
Let's make sure we add this to the .gitignore so we don't have any issues with our deployments

Reply:
Yeah for sure